extensions/registry.py

```diff
@@ -2314,6 +2314,12 @@
         'supporters' : { 'INTEL', 'NVIDIA' },
         'url' : 'extensions/EXT/EXT_texture_compression_s3tc.txt',
     },
+    'GL_EXT_texture_compression_s3tc_srgb' : {
+        'esnumber' : 286,
```

Comment:
I believe the next available ES number is 289 (please run 'python nextfree.py' in the extensions folder to find the next available number.